compiler/front/main.nim

```diff
@@ -427,10 +424,3 @@ proc mainCommand*(graph: ModuleGraph) =
       echo conf.dump(conf.vmProfileData)
     genSuccessX(conf)
 
-  when PrintRopeCacheStats:
```

Comment:
I did a quick search and it seems there are some global variables left around `gCacheTries` for one.